crm_rma_prodlot_invoice/README.rst

```diff
@@ -0,0 +1,72 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+CRM Claim Prodlot Invoice
+=========================
+
+When an invoice is created from sale order, picking and procurement information
+is kept into invoice line.
+
+Also, helps to display invoice line lot information when an invoice line is displayed
```

Comment:
Ok, this documentation will improve. Thank you very much for your remark.

@osvalr and me we will meet today for this.